doc/languages-frameworks/go.xml

```diff
@@ -152,7 +152,7 @@ deis = buildGoPackage rec {
     </callout>
     <callout arearefs='ex-buildGoPackage-5'>
      <para>
-      <varname>buildFlags</varname> is a list of flags passed to the go build command.
+       <varname>buildFlags</varname> is a list of flags passed to the go build command, explained with more examples <link xlink:href="#ssec-go-common-attributes">here</link>.
```

Comment:
I'd remove this line and line 122 entirely and just have "Attributes used by the builders" section otherwise it still looks to be specific to `buildGoPackage`.

Later we could de-dupe `subPackages`, `deleteVendor` and add other common attrs (e.g. `excludedPackages`).

Reply:
I agree.

Reply:
I removed the commonly respected attributes from the examples to a common section where all of them are explained.